docs/static/releasenotes.asciidoc

```diff
@@ -30,21 +30,22 @@ This section summarizes the changes in the following releases:
 [[logstash-7-0-0-beta1]]
 === Logstash 7.0.0-beta1 Release Notes
 
-Placeholder for release notes
+* Update Java dependencies https://github.com/elastic/logstash/pull/10340[#10340]
+* Remove pipeline output workers setting https://github.com/elastic/logstash/pull/10358[#10358]
+* Cleanup Ruby gems dependencies https://github.com/elastic/logstash/pull/10171[#10171]
+* Ensure compatibility of module data with ES and Kibana 7.0 https://github.com/elastic/logstash/pull/10356[#10356]
+* Rename x-pack monitoring and management config option .url and .ca to .hosts and .certificate_authority https://github.com/elastic/logstash/pull/10380[#10380]
+* BUGFIX: building of deb and rpm artifacts https://github.com/elastic/logstash/pull/10396[#10396]
 
 [[logstash-7-0-0-alpha2]]
 === Logstash 7.0.0-alpha2 Release Notes
 
-Placeholder for release notes
+* No user-facing changes
```

Comment:
I think we can add here at least:

```
* d5b5f9a4f - Add Zing JVM to the list of supported garbage collectors. (4 months ago)
* e4159d0fb - BUGFIX: setting a list config to empty array should not result in nil (4 months ago)
```

Reply:
Are you sure? https://github.com/elastic/logstash/pull/10051 was also merged into 6.x and https://github.com/elastic/logstash/pull/10179 in 6.5 and 6.x
I am a bit confused about what we should add in these release notes; I omitted all that were already previously merged into a v6 branch ...

Reply:
Ah, I see, what you say makes sense. Nevermind then